backend/src/main/java/com/redhat/cloud/notifications/db/repositories/SubscriptionRepository.java

```diff
@@ -51,7 +51,7 @@ public void unsubscribe(String orgId, String username, UUID eventTypeId, Subscri
     @Transactional
     void updateSubscription(String orgId, String username, UUID eventTypeId, SubscriptionType subscriptionType, boolean subscribed) {
 
-        if (subscribed) {
+        if (subscribed && !backendConfig.isUseCommonTemplateModuleForUserPrefApisToggle()) {
```

Comment:
Is it okay to remove this check? Can't we run it with the files instead of the DB?